roles/edpm_iscsid/tasks/run.yml

```diff
@@ -23,16 +23,22 @@
   ansible.builtin.include_tasks:
     file: healthchecks.yml
 
-- name: Manage iscsid containers
-  ansible.builtin.include_role:
-    name: edpm_container_standalone
+- name: Cleanup existing standalone conatiner
+  ansible.builtin.import_role:
+    name: edpm_container_quadlets
+    tasks_from: cleanup.yml
+  vars:
+    edpm_container_quadlet_service: iscsid
```

Comment:
this var should be defined in the new role's defaults/main.yml, even with an empty value. And I suggest calling the new role just `edpm_container_quadlet` (without an 's').

Reply:
ack

Reply:
edpm_container_quadlet/tasks/cleanup.yml needs to be changed to:

`
edpm_{{ edpm_container_quadlet_service }}
`

in order for it to find the servie to stop